intezer_sdk/api.py

```diff
@@ -254,6 +365,16 @@ def get_endpoint_sub_analyses(self, analyses_id: str, verdicts: Optional[List[st
 
         return response.json()['sub_analyses']
 
+    def  create_endpoint_scan(self, scanner_info: dict) -> Dict[str, str]:
```

Comment:
this is probably here by mistake

Reply:
no, scanner first creates the scan with the normal api (unversioned) and then moves to communicating with the endpoint scan api